src/test/scala/akka/wamp/ManagerSpec.scala

```diff
@@ -56,7 +58,46 @@ class ManagerSpec
     pending
   }
 
-  it should "Fail on wrong message format" in { f =>
+  it should "Receive text messages and emit Events" in { f =>
```

Comment:
This ``ManagerSpec`` test suite is meant to put under test those few behaviours the ``IO(Wamp)`` (a.k.a. the manager) is responsible of: bind/unbind a router and connect/disconnect a client.

I'm sorry I'll have to move your additional tests (especially those testing publish/subscriber messaging) to some other tests suite.